WordPress/gutenberg
# Change edit links for templates and template parts

## Description

Close https://github.com/WordPress/gutenberg/issues/35990.

Edit the "Edit" link of the items in Templates and Template Parts list to point to Site Editor.

## How has this been tested?



1. Enable block based theme like `tt1-blocks`
2. Go to Appearance -> Templates and Appearance -> Template parts
3. The item should have edit links pointing to the site editor urls

## Types of changes




Bug fix

## Checklist:
